app/lib/reducers/ui.js

```diff
@@ -24,7 +24,15 @@ const initial = Immutable({
   fontFamily: 'Menlo, "DejaVu Sans Mono", "Lucida Console", monospace',
   fontSizeOverride: null,
   css: '',
-  termCSS: '',
+  termCSS: '' +
+    'x-row {' +
+    '  text-shadow: 0 0 0 #fff;' +
```

Comment:
I went with css variables for the colours, eventually I think we should move all colours to css variables, if not for anything else it makes easier to develop themes, you can change them right in the inspector